IMDA-BTG/aiverify
# update poetry requirements with python311 support

## Description

Update on library versions (generated by poetry) to allow local testing with pyenv and tox, mainly to test for the plugins compatibility with Python 3.11.

While running the plugins in the backend locally, our observation is that all plugins are able to run with Python3.11. 

## Motivation and Context

Test plugins with Python 3.11 to see if they can be run successfully.

## Type of Change



## How to Test

Run python . in each of the plugin's dir (where __main__.py is) where Python's version is 3.11.
* locally the team uses pyenv and tox to easily test with various Python versions but the configuration files will not be pushed as it is not really relevant to running AI Verify 

## Checklist

Please check all the boxes that apply to this pull request using "x":

- [x ]  I have tested the changes locally and verified that they work as expected.
- [x ]  I have added appropriate unit tests or functional tests for the changes made.
- [x ]  I have followed the project's coding conventions and style guidelines.
- [x ]  I have performed a self-review of my own code.

## Screenshots (if applicable)

-

## Additional Notes

-
